system/Email/Email.php

```diff
@@ -820,7 +820,6 @@ protected function getMessageID()
 	 */
 	protected function getProtocol()
 	{
-		$this->protocol                                                      = strtolower($this->protocol);
```

Comment:
Verify that this is set lowercase elsewhere. Otherwise, we do need to keep this in.